### PR TITLE
fix: Fix ImageBrush.Stretch on Wasm

### LINKED ISSUE #3621: [Wasm] ImageBrush Stretch property is not applied
## Current behavior

ImageBrush Stretch property is not applied (Uniform, Fill and UniformToFill)

## Expected behavior

ImageBrush Stretch property should be applied

## How to reproduce it (as minimally and precisely as possible)

Sample:
[ImageBrushSample.zip](https://github.com/unoplatform/uno/files/4972877/ImageBrushSample.zip)

## Environment

Nuget Package:
Uno.UI
Uno.UI.WebAssembly

Package Version(s):
3.0.0-dev.956

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [x] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?

No internal issue

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #3621

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Media/ImageBrush.wasm.cs [hunk 1]
```diff
@@ -43,8 +43,8 @@ internal string ToCssBackgroundSize()
 			{
 				Stretch.Fill => "100% 100%",
 				Stretch.None => "auto",
-				Stretch.Uniform => "auto", // patch for now
-				Stretch.UniformToFill => "auto", // patch for now
+				Stretch.Uniform => "contain", // patch for now
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/ImageBrush.wasm.cs:46]: This doesn't exactly match UWP, but is the most similar to UWP.

**Comment by jeromelaban** [comment]
Thanks for your contribuion! 

Can you add a UI test validation for this behavior ?

**Comment by Youssef1313** [comment]
Test is skipped on Android for now. `Stretch.None` is broken:

image

**Comment by jeromelaban** [comment]
Let's revalidate the screenshot tests results from the CI.

**Comment by Youssef1313** [comment]
> Let's revalidate the screenshot tests results from the CI.

@jeromelaban Let me know if there is something I need to do here :)

**Review by carldebilly — APPROVED** [review]

**Comment by jeromelaban** [comment]
Looks like it's definitely better: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=38079&view=ms.vss-test-web.build-test-results-tab&runId=1661084&resultId=100905&paneView=attachments :)

Thanks @Youssef1313!

**Pull request merged**